Tests/APITesters/SwiftAPITester/SwiftAPITester/PresentedOfferingContextAPI.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  PresentedOfferingContextAPI.swift
+//  SwiftAPITester
+//
+//  Created by Josh Holtz on 2/14/24.
+//
+
+import Foundation
+import RevenueCat
+import StoreKit
+
+private var context: PresentedOfferingContext!
+func checkPresentedOfferingContextAPI() {
+    let oID: String = context.offeringIdentifier
+
+    print(context!, oID)
+}
+
+private func checkCreatePresentedOfferingContextAPI() {
+    _ = PresentedOfferingContext(offeringIdentifier: "")
```

Comment:
Nit: this could be an optional `init?` initializer. To prevent that false positive:
```suggestion
    let _: PresentedOfferingContext = .init(offeringIdentifier: "")
```